webapp/webpack.config.js

```diff
@@ -17,7 +17,7 @@ if (NPM_TARGET === 'debug' || NPM_TARGET === 'debug:watch') {
 
 const plugins = [
     new webpack.ProvidePlugin({
-        process: 'process/browser',
+        process: 'process/browser.js',
```

Comment:
Hope this wasn't too painful to figure out :grimacing:

Reply:
It was touch and go there for a bit (my sanity)